openapi/resource-server.yaml

```diff
@@ -684,9 +686,6 @@ paths:
         '404':
           description: Incoming Payment Not Found
       description: A client can fetch the latest state of an incoming payment to determine the amount received into the payment pointer.
-      parameters:
-        - $ref: '#/components/parameters/signature-input'
-        - $ref: '#/components/parameters/signature'
```

Comment:
I tried to make these optional since they are still required to get the full incoming-payment details but it looks like this level of parameter (under `/get`) is required, which is why I removed them: https://stackoverflow.com/questions/35011192/how-to-define-an-optional-parameter-in-path-using-swagger

Reply:
You could remove the `required: true` property from each of the signature components in the [shared components part of the spec](https://github.com/interledger/open-payments/blob/02ef4679da695fa47c32120309e73b4312c3dcab/openapi/resource-server.yaml#L1365-L1380), but that would make it optional for all resource paths...

Maybe there could be a one-off definition of the Signature parameters for this url that are optional. Or we could add `optional-signature` and `optional-signature-input` in the components section that are duplicates of `signature` and `signature-input` that optional requirements.

Reply:
I added the `optional-signature` paramaters